content/posts/k0s-1.34-released/index.md

```diff
@@ -0,0 +1,99 @@
+- **Networking Enhancements:** Direct Server Return (DSR) support in Windows kube-proxy for reduced latency and load on the balancer.
+- **Volume and Storage Updates:** VolumeAttributesClass GA and recovery from volume expansion failures improve flexibility and reliability.
+- **Pod-level Resource Management (Beta):** Define total Pod resources shared across multiple containers, simplifying scheduling.
+- **Container Lifecycle Hooks:** Stable Sleep action for PreStop/PostStart hooks to improve lifecycle management.
+- **Streaming and Caching Improvements:** Streaming LIST responses and resilient watch cache initialization reduce memory pressure and improve API server performance.
+- **Security Enhancements:** Pod certificates for mTLS authentication (Alpha), mutating admission policies (Beta), and stricter anonymous request controls.
+
+Refer to the [Kubernetes 1.34 release notes](https://kubernetes.io/blog/2025/08/27/kubernetes-v1-34-release/) for details.
+
+---
+
+## What’s New in k0s 1.34
+
+This release focuses on four key areas: IPv6 networking, Windows stability and integration, feature gating, and developer automation.
+
+---
+
+### Windows Node Improvements
+
+Windows support continues to mature in this release, improving node lifecycle, process management, and cluster consistency.
+
+- Refined **kubelet process cleanup** during node shutdown and restart sequences to prevent stale processes.
+- Enhanced **Windows service stop handling**, ensuring kubelet exits cleanly under SIGTERM-equivalent events.
+- Added **semantic kubelet configuration validation** for Windows worker profiles, preventing invalid kubelet settings from applying.
+- Fixed **path resolution issues** in worker runtime directory initialization, ensuring reliable setup on Windows hosts.
+- Improved **CRI runtime connection management** for Windows containers, reducing transient startup failures.
+- Validated **dual-stack networking configurations** for mixed Linux/Windows clusters.
+- Ensured **Calico Windows components** align with new IPv6 and security context changes.
+- Improved **PowerShell-based packaging scripts** for agent binaries and runtime dependencies.
+- Strengthened **cross-platform conformance testing**, ensuring parity between Linux and Windows node behavior.
+
+These changes significantly improve reliability for Windows-based worker nodes and pave the way for smoother hybrid cluster operations.
+
+### IPv6 Single-Stack
+
+K0s 1.34 introduces alpha IPv6 only clusters for modern or constrained deployments. Previously k0s only supported IPv4 single-stack or
+IPv4 and IPv6 dual-stack. All k0s features are implemented in IPv6 single stack, including both calico and kube-router support, NLLB, CPLB
+among other features.
+
+Learn more in our [IPv6 Single-Stack documentation](https://docs.k0sproject.io/v1.34.1+k0s.0/single-stack-ipv6/).
```

Comment:
Should we maybe point to `stable` alias docs link? The problem with versioned links is that Google and others remembers those too well...